src/gridtypes/ParaviewCollection.jl

```diff
@@ -20,7 +20,7 @@ function collection_add_timestep(pvd::CollectionFile, datfile::VTKFile,
     xroot = root(pvd.xdoc)
     xMBDS = find_element(xroot, "Collection")
     xDataSet = new_child(xMBDS, "DataSet")
-    fname = splitdir(datfile.path)[2]
+    fname = relpath(datfile.path |> abspath ,pvd.path |> abspath |> splitdir |> first)
```

Comment:
Maybe
```suggestion
    fname = relpath(abspath(datfile.path), first(splitdir(abspath(pvd.path))))
```
to follow the style of the rest of the code? :)

Reply:
Perfect, thanks!

Reply:
Also fine. I just like the clean look you get when using `|>`

Reply:
> I just like the clean look you get when using |>

Well, I like the clean look when you don't use it :)